src/components/content/conditions/patient-conditions-actions.tsx

```diff
@@ -5,60 +5,67 @@ import { useAddConditionForm } from "./condition-hooks";
 import { FilterCollection } from "./patient-conditions-filters";
 import { Sort, SortOption } from "./patient-conditions-sort";
 import { DropdownMenuAction } from "@/components/core/dropdown-action-menu";
-import { Toggle } from "@/components/core/toggle";
+import { FilterBar } from "@/components/core/filter-bar/filter-bar";
+import {
+  FilterChangeEvent,
+  FilterItem,
+} from "@/components/core/filter-bar/filter-bar-types";
 
 export type PatientConditionsActionsProps = {
   hideAdd: boolean;
-  onToggleShowHistoric: () => void;
   sortOptions: SortOption[];
   updateSorts: (newSorts: Partial<Sort>) => void;
   activeCollection: FilterCollection;
   currentSorts: Sort;
+  filterItems: FilterItem[];
+  setFilters: (filterChangeEvent: FilterChangeEvent) => void;
+  filters: FilterChangeEvent;
 };
 
 export function PatientConditionsActions({
   hideAdd,
-  onToggleShowHistoric,
   sortOptions,
   updateSorts,
   activeCollection,
   currentSorts,
+  filterItems,
+  setFilters,
+  filters,
 }: PatientConditionsActionsProps) {
   const showAddConditionForm = useAddConditionForm();
   const patientHistory = usePatientHistory();
 
   return (
-    <div className="ctw-flex ctw-items-center ctw-justify-between ctw-space-x-2 ctw-p-3 sm:ctw-pt-1.5">
-      {patientHistory.lastRetrievedAt && hideAdd && (
-        <PatientHistoryTableHeaderMessage
-          patientHistory={patientHistory}
-          message="Last Retrieved"
-        />
-      )}
-      <div>
+    <div className="ctw-flex ctw-items-center ctw-justify-between ctw-space-x-2 ctw-p-3">
```

Comment:
The `sm:ctw-pt-1.5` class attempts to make the x-space between tabs -> actions -> content even on larger displays
```suggestion
    <div className="ctw-flex ctw-items-center ctw-justify-between ctw-space-x-2 ctw-p-3 sm:ctw-pt-1.5">
```

With:
![Screen Shot 2023-02-13 at 12 47 20 PM](https://user-images.githubusercontent.com/6380075/218533472-3025b46b-ad51-41af-aa19-8ed05b77ed87.png)

Without _(extra vertical white space above actions):
![Screen Shot 2023-02-13 at 12 47 54 PM](https://user-images.githubusercontent.com/6380075/218533607-1fbe62c6-09b8-4154-a552-8de1edcd4b4b.png)